orchestrator/graphql/schemas/product_block.py

```diff
@@ -15,4 +15,12 @@ class ProductBlock:
     status: strawberry.auto
     created_at: strawberry.auto
     end_date: strawberry.auto
-    resource_types: Optional[list[ResourceType]]
+    resource_types: list[ResourceType]
+
+    @strawberry.field(description="Return all product blocks that this product block depends on")  # type: ignore
+    async def depends_on(self) -> List[Annotated["ProductBlock", strawberry.lazy(".product_block")]]:
```

Comment:
Mix of list and List?

Reply:
thought something would complain about it...

Reply:
changed to `list`